backend/src/main/java/org/example/spring/boot/skeleton/config/WebSecurityConfig.java

```diff
@@ -16,32 +17,36 @@
 import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
 
 @Configuration
-@AllArgsConstructor
 public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
-
+    @Autowired
     private JwtAuthenticationEntryPoint authenticationEntryPoint;
+    @Autowired
     private UserDetailsService userDetailsService;
+    @Autowired
     private JwtFilter filter;
 
+    @Value(value = "${admin}")
```

Comment:
ez lehetne pontosabb, hogy amikor valaki elolvassa a properties file-t, akkor kód átnézése nélkül is könnyedén kitalálja, hogy milyen admin?